ocs_ci/templates/fusion-aas/managedfusionoffering.yaml.j2

```
@@ -6,6 +6,6 @@ metadata:
 spec:
   kind: DF
   release: "{{ ocp_version }}"
-  config: 
+  config: |
     usableCapacityInTiB: "{{ size }}"
```

```suggestion
    usableCapacityInTiB: {{ size }}
```
We no longer require "" for size param Sample for [DF](https://github.com/red-hat-storage/managed-fusion-agent/blob/main/config/samples/misf_v1alpha1_managedfusionoffering.yaml)

---

Removed the quotes.